### PR TITLE
1.0  Added and updated /container-deployment docker related info

### PR DESCRIPTION
**Pull request by digikin**
ref: #75

There isn't Container Deployment in v1.0 doc. but 0.12.0 document has it https://docs.fluentd.org/v/0.12/container-deployment

Added and updated /container-deployment
Only changed the docker related part of the documentation.  I am still researching the kubernetes update.  

Let me know if anything needs to be changed.

### CODE REVIEW

**Comment by repeatedly** [comment]
@ganmacs Could you check this?

File: container-deployment/docker-compose.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+![](/images/7.2_efk-kibana-discover.png)
+
+## Conclusion
+
+This article explains how to collect logs from Apache to EFK
+(Elasticsearch + Fluentd + Kibana). The example code is available in
+this repository.
+-   <https://github.com/digikin/fluentd-elastic-kibana>
+-   <https://github.com/kzk/docker-compose-efk>
```
Comment (ganmacs) [inline container-deployment/docker-compose.md]: could you remove this line?

Comment (digikin) [reply container-deployment/docker-compose.md]: Are you wanting me to remove line 194 and 195? I made a repo that contained all the example code so it was easy to pull the example?

Comment (ganmacs) [reply container-deployment/docker-compose.md]: No. Only L195. L195 is not related with this article.

**Comment by ganmacs** [comment]
thnaks!

**Pull request merged**